tssc/factory.py

```diff
@@ -32,14 +35,16 @@ class TSSCFactory:
     _step_implementers = {}
 
     def __init__(self, config, results_dir_path='tssc-results', \
-      results_file_name='tssc-results.yml'):
+            results_file_name='tssc-results.yml', \
+            work_dir_path='tssc-working'):
```

Comment:
@kfrankli this is where that working dir name has default value set

Reply:
Whoops I meant to remove the comment as I found it. That file was the first one I saw, so I hadn't figured out what working dir was yet.